client/state/utils/schema-utils.js

```diff
@@ -105,31 +105,26 @@ export const withSchemaValidation = ( schema, reducer ) => {
 		throw new Error( 'null schema passed to withSchemaValidation' );
 	}
 
-	const wrappedReducer = ( state, action ) => {
-		if ( action.type === DESERIALIZE ) {
```

Comment:
Let's also update the JSdoc above - specifically it contains some examples with the old action types.